src/utils/jwt/jwt.test.ts

```diff
@@ -339,7 +339,10 @@ describe('JWT', () => {
     }
     const secret = await crypto.subtle.importKey(
       'raw',
-      Buffer.from('cefb73234d5fae4bf27662900732b52943e8d53e871fe0f353da95de4599c21d', 'hex'),
+      Buffer.from(
+        'cefb73234d5fae4bf27662900732b52943e8d53e871fe0f353da95de4599c21d',
+        'hex'
+      ) as BufferSource,
```

Comment:
`Buffer.from()` returns a Node.js Buffer whose type doesn't directly match `BufferSource` in TypeScript 5.9.